README.md

```diff
@@ -1582,6 +1583,34 @@ In your other local project directory:
 yarn link @okta/okta-signin-widget
 ```
 
+### Utilizing Pseudo-loc
+
+> :warning: This tool requires access to Okta's internal registry via the VPN.
+
+A pseudo-localized language is a test language created to identify issues with the internationalization process. Generated from `login.properties` English resources, the pseudo-loc properties file can be used to test UI's for English leaks and CSS layout issues caused due to localization.
+
+To generate pseudo-loc, run the following command:
+
+```sh
+# Navigate into the pseudo-loc package
+[okta-siginin-widget]$ cd packages/@okta/pseudo-loc/
+
+# Install all required dependencies and generate login_ok_PL.propertiies
+# NOTE: This requires VPN access
```

Comment:
Is it possible to return a warning/error that a VPN is required for this on command prompt?

Reply:
@pradeepdewda-okta - We do. When the [install command fails](https://github.com/okta/okta-signin-widget/blob/3bc9c1c1cfbc55b4dfc186d0ebe8777945bd4476/buildtools/commands/pseudo-loc.js#L30).